src/archival/client.rs

```diff
@@ -0,0 +1,31 @@
+use crate::configuration::Settings;
+use once_cell::sync::Lazy;
+use reqwest::{header, Client};
+use std::sync::Arc;
+
+pub static REQWEST_CLIENT: Lazy<Arc<Client>> = Lazy::new(|| {
```

Comment:
reqwest clients are already an `Arc` over their internals. You don't need to wrap them in one. Just clone it directly

https://docs.rs/reqwest/latest/reqwest/struct.Client.html

https://docs.rs/reqwest/latest/src/reqwest/async_impl/client.rs.html#70

Reply:
Ohh cool, will update it

Reply:
updated ✅  https://github.com/yellowHatpro/mb-exurl-ia-service/commit/e3294e8ca00d5254c0f0461b481a396af6e873af